ql/src/java/org/apache/hadoop/hive/ql/io/AcidUtils.java

```diff
@@ -2265,6 +2268,50 @@ public static Boolean isToInsertOnlyTable(Table tbl, Map<String, String> props)
     return true;
   }
 
+  public static Boolean isToFullAcid(Table table, Map<String, String> props) {
+    String transactional = props.get(hive_metastoreConstants.TABLE_IS_TRANSACTIONAL);
```

Comment:
why can we simply use AcidUtils.isTransactionalTable(props)/AcidUtils.isTransactionalTable(table.getParameters())?